src/modules/Comment/validation.ts

```diff
@@ -0,0 +1,14 @@
+import Joi from 'joi';
+
+const comments = {
+  body: Joi.object({
+    title: Joi.string().required(),
+    description: Joi.string().required(),
+    source: Joi.string().required(),
+    url: Joi.string().required(),
```

Comment:
hey @bojana-petroska i would also add that url should be valid. By adding .uri() -> this validates  that the string is a valid URI